src/stories/Demo/Demo.stories.js

```diff
@@ -210,6 +210,7 @@ export const demo = () => {
         readOnly={readOnly}
         ref={refUse}
         // loadTemplateObject={fetchTemplateObj}
+        onClauseUpdated={() => {console.log('Demo')})}
```

Comment:
There's an extra `)` between the curlies

Reply:
👀